pkgs/development/python-modules/pypika/default.nix

```diff
@@ -0,0 +1,31 @@
+{  lib
+,  buildPythonPackage
+,  fetchFromGitHub
+,  parameterized
+}:
+buildPythonPackage rec {
+  pname = "pypika";
+  version = "0.48.9";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "kayak";
+    repo = "pypika";
+    rev = "v${version}";
+    hash = "sha256-9HKT1xRu23F5ptiKhIgIR8srLIcpDzpowBNuYOhqMU0=";
+  };
+
+  pythonImportsCheck = ["pypika"];
+
+  nativeCheckInputs = [
```

Comment:
Would you please add `unittestCheckHook` to enable the test?

Reply:
Thanks for the tip. TIL

Reply:
How do I know that this needs to be set? Is there any tell-tale sign in the upstream code?

Reply:
Ok, added them. running `nixpkgs-review` again, now.

Reply:
Basically, we can use either `pytestCheckHook` or `unittestCheckHook`.
It is often possible to find out which one to use by looking at the definition in, for example, tox.ini or ci.

If the upstream has no tests, we should set `doCheck = false;` with the reason.

Reply:
Ok, in this case [we have `import unittest`](https://github.com/kayak/pypika/blob/master/pypika/tests/test_aggregate.py#L1).  So that was the right choice, then.

Thx for the help!